src/archetype.rs

```diff
@@ -494,7 +494,10 @@ impl<V> OrderedTypeIdMap<V> {
     }
 }
 
-/// Metadata required to store a component
+/// Metadata required to store a component. All told, this means a [`TypeId`], to be able to
+/// dynamically name/check the component type; a [`Layout`], so that we know how to allocate memory
```

Comment:
There should be an empty line here after the first sentence.

Reply:
Added.